src/fix/main.c

```diff
@@ -127,6 +127,17 @@ enum MbcType {
 	MBC_BAD_RANGE, // MBC number out of range
 };
 
+static void printAcceptedMBCNames(void)
```

Comment:
Imo 28 lines is a lot... I'll experiment with it.

Reply:
With the coming addition of TPP1, this is gonna grow to 44 lines, and to a full 60 if they were to add another feature. Imo MBCs should be grouped on separate lines, though I will play with this a bit, as the MBC3 and MBC5 lines are too long.

Reply:
Or, don't have the partial grouping of MBCs on lines, and just wrap at a max length.

```c
puts("ROM, ROM+RAM, ROM+RAM+BATTERY, MBC1, MBC1+RAM, MBC1+RAM+BATTERY, MBC2,");
puts("MBC2+BATTERY, MMM01, MMM01+RAM, MMM01+RAM+BATTERY, MBC3, MBC3+TIMER+BATTERY,");
puts("MBC3+TIMER+RAM+BATTERY, MBC3+RAM, MBC3+RAM+BATTERY, MBC5, MBC5+RAM,");
puts("MBC5+RAM+BATTERY, MBC5+RUMBLE, MBC5+RUMBLE+RAM, MBC5+RUMBLE+RAM+BATTERY,");
puts("MBC6, MBC7+SENSOR+RUMBLE+RAM+BATTERY, POCKET_CAMERA, BANDAI_TAMA5, HUC3,");
puts("HUC1+RAM+BATTERY"); // plus TPP1 variations later
```

Reply:
Here is a middle ground I came up with; it would also be possible the last four, imo.
![](https://media.discordapp.net/attachments/661193788802203688/838175569139990528/unknown.png)

Grouping the names helps readability a lot, imo, and that's why I'm trying to make it work.

Reply:
`ROM ($00)` could be `ROM ($00) aka ROM_ONLY`